layers/+source-control/git/README.org

```diff
@@ -136,6 +159,46 @@ To enable it you have to add the following lines to your
 #+END_SRC
 
 ** Forge
+Magit Forge can view and create issues & pull requests with forges
+(e.g. GitHub, GitLab)
+
+Magit Forge requires a username for the respective forge and will prompt for a
+username if not found, writing it to  =~/.gitconfig=
+
+Explicitly define a forge identity using the `git` command.
+
+For GitHub:
+
+#+begin_src shell
+  git config --global github.name "username"
+#+end_src
+
+For GitLab:
+
+#+begin_src shell
+  git config --global gitlab.name "username"
+#+end_src
+
+See the official [[https://magit.vc/manual/forge/Getting-Started.html#Getting-Started][Magit Forge]] and [[https://magit.vc/manual/ghub/Getting-Started.html][GHub Getting Started]] for general guides or follow
+a community written [[https://practical.li/spacemacs/source-control/forge-configuration.html][Spacemacs specific guide to configuring Magit Forge]].
+
+*** Magit Forge configuration
+For each forge (e.g. GitHub, GitLab), add a machine configuration to the
+=~/.authinfo= or PGP encrypted =~/.authinfo.gpg= file (for increased security).
```

Comment:
I think we should only mention .authinfo.pgp here. Because .authinfo is insecure.

User may find the insecure way and decide use it anyway, if they've actually read Emacs manual, but hen that's out of our responsibility. :smile: